### PR TITLE
Update AvaritiaPatches.java

### PR DESCRIPTION
**Pull request by EnderProyects**
add recipe for the creative flower using all mana flowers

### CODE REVIEW

**Review by combusterf — CHANGES_REQUESTED** [review]
- There already exists a recipe for the Asgardandelion
- This new recipe is an order of magnitude more expensive than the ones for the creative pool and creative tablet, despite having equivalent functionality

**Comment by EnderProyects** [comment]
I make the pr because I have it in my computer for some months for now, also I make it for two reasons:
- With this recipe people NEED to make at least something of botania in order to get the flower, in the normal recipe you get free mana for nothing of content of botania
- For me it was like giving people infinite amount of essentia without using thaumcraft
P.s I can change the infinite plates amount needed for the recipe if you like it

**Comment by EnderProyects** [comment]
Also in terms of costs the original recipe is already expensive about inf ingots ( but without any botania cost)
image

**Comment by EnderProyects** [comment]
And here you have the one I propossed months ago it uses a litle bit more of ingots in plate form, and all the mana producer flowers:
image

**Review by combusterf — CHANGES_REQUESTED** [review]
Argument ad nauseam. Previous points were NOT addressed, not even a commit has been added.

**Comment by combusterf** [comment]
- Infinity is still expensive, using dense plates over single ingots is just bullying the user and an abuse of server resources. 
- Once you hit late UHV+ and get infinity you are entitled to braindead solutions. Your average player gets into Botania 10 tiers earlier than when they can make the infinity stuff.
- Adding every flower is also just about bullying the user into useless specific crafting effort rather than testing that they have infrastructure.
- The previous consensus in Discord was to not include stuff outside of botania - looking specifically at that thaintthistle that requires a bunch of thaumcraft nuisance.

**Comment by EnderProyects** [comment]
image
The thaintthistle is an eassy recipe, I dont see where is the problem, I can substitute it with the Gourmaryllis ( that I didnt include because is used in the craft of the Soarleander)
Also what about substituing the central dense plates with various plates of the mod itself? 
image

**Comment by Dream-Master** [comment]
@EnderProyects any news ?

**Comment by EnderProyects** [comment]
I'm at work, I will pr this later

**Comment by EnderProyects** [comment]
I have messed something up in the recipe

**Comment by EnderProyects** [comment]
I still have an issue, that I dont know what is happening, since all seems correct:
This turn into fire, instead of the infinitato
'Z', new ItemStack(LudicrousBlocks.infinitato),

File: src/main/java/net/fuzzycraft/botanichorizons/patches/AvaritiaPatches.java [hunk 1]
```diff
@@ -113,7 +113,8 @@ public static void applyPatches() {
                 '3', new ItemStack(ModItems.blackLotus,1,1),
                 'S', ItemBlockSpecialFlower.ofType(LibBlockNames.SUBTILE_SPECTROLUS),
                 'm', ItemBlockSpecialFlower.ofType(LibBlockNames.SUBTILE_NARSLIMMUS),
-                'Z', new ItemStack(LudicrousBlocks.infinitato),
+                //'Z', new ItemStack(LudicrousBlocks.infinitato), // Until is fixed
```
Comment (Dream-Master) [inline src/main/java/net/fuzzycraft/botanichorizons/patches/AvaritiaPatches.java:116]: fix it first. No duct tape recipes

**Review by Dream-Master — CHANGES_REQUESTED** [review]

**Comment by Dream-Master** [comment]
@EnderProyects any news here ?

**Comment by EnderProyects** [comment]
I tried many things and asked to people where the error is and haven't find a solution, so ATM the current recipe ( of the last pr) is what I would like to pr

**Comment by Dream-Master** [comment]
i set it draft until it is fixed

**Pull request closed without merging**